jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/signal/ProcessInstanceSignalPresenter.java

```diff
@@ -94,6 +99,7 @@ public void signalProcessInstances(List<Long> processInstanceIds) {
         }
         processService.call((Void v) -> {
                                 processInstancesUpdatedEvent.fire(new ProcessInstancesUpdateEvent(0L));
+            bulkActionPerformedEvent.fire(new BulkActionPerformedEvent());
```

Comment:
@jstastny-cz here!

Reply:
... I didn't consider this affecting task bulk actions due to the class name ... so the ProcessInstanceSignalPresenter is also taking part in task bulk actions?

Reply:
Nop, It's performing a multiple signaling on process instances that requires introduction of data: name and data. It had the same issue with the checkboxes you found at task. That's the reason to create this generic 'BulkActionPerformedEvent' to be able to deselect when the bulk action is performed

Reply:
@jstastny-cz maybe makes sense rename the JBPM jira and the commit to not to be linked with task bulk operation, and just talk about bulk actions and checkboxes?

Reply:
that's what I assumed, but then I could not see a place where for tasks the event is being fired other than during reassign operation.

Reply:
yes, this event is only used when the operation requires  introduction of data (signal data in case of PI and user id in case of task reassignment). The other, have bulk action confirmation that includes the unselection once it's confirmed

Reply:
Wouldn't be unified solution (firing the event everywhere) better? Having two different behaviors for that same sort of actions is puzzling (example: me).